easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.3.0-foss-2017b-METIS-5.1.0.eb

```diff
@@ -0,0 +1,32 @@
+name = 'SuiteSparse'
+version = '5.3.0'
+metis_ver = '5.1.0'
+versionsuffix = '-METIS-%s' % metis_ver
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['90e69713d8c454da5a95a839aea5d97d8d03d00cc1f667c4bdfca03f640f963d']
+
+builddependencies = [
+    ('CMake', '3.9.5', '', ('GCCcore', '6.4.0')),
+]
```

Comment:
@HPC-UniOldenburg There's no need to hardcode the (sub)toolchain here for `CMake` (and it's actually harmful since it complicates future updates):

```python
builddependencies = [('CMake', '3.9.5')]
```